src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type CommentInfoItem = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<CommentInfoItem> = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
```

Comment:
Consider using CSS for creating space between elements. You can wrap it inside the span element with the appropriate class name and add padding or margin here